packages/engine/src/tests/test_p2ms.cairo

```diff
@@ -0,0 +1,106 @@
+use crate::transaction::TransactionTrait;
+use crate::utxo::UTXO;
+use crate::validate;
+use shinigami_utils::bytecode::hex_to_bytecode;
+
+#[test]
+fn test_p2ms_1_of_2() {
+    // First P2MS transaction made on 30 Jan 2012
+    // tx: 60a20bd93aa49ab4b28d514ec10b06e1829ce6818ec06cd3aabd013ebcdc4bb1
+    let prevout_pk_script =
+        "0x514104cc71eb30d653c0c3163990c47b976f3fb3f37cccdcbedb169a1dfef58bbfbfaff7d8a473e7e2e6d317b87bafe8bde97e3cf8f065dec022b51d11fcdd0d348ac4410461cbdcc5409fb4b4d42b51d33381354d80e550078cb532a34bfa2fcfdeb7d76519aecc62770f5b0e4ef8551946d8a540911abe3e7854a26f39f58b25c15342af52ae";
+    let prev_out = UTXO {
+        amount: 1000000, pubkey_script: hex_to_bytecode(@prevout_pk_script), block_height: 164467
+    };
+
+    let raw_transaction_hex =
+        "0x0100000001b14bdcbc3e01bdaad36cc08e81e69c82e1060bc14e518db2b49aa43ad90ba26000000000490047304402203f16c6f40162ab686621ef3000b04e75418a0c0cb2d8aebeac894ae360ac1e780220ddc15ecdfc3507ac48e1681a33eb60996631bf6bf5bc0a0682c4db743ce7ca2b01ffffffff0140420f00000000001976a914660d4ef3a743e3e696ad990364e555c271ad504b88ac00000000";
+    let raw_transaction = hex_to_bytecode(@raw_transaction_hex);
+    let transaction = TransactionTrait::deserialize(raw_transaction);
+    let utxo_hints = array![prev_out];
+
+    let res = validate::validate_p2ms(@transaction, 0, utxo_hints);
+    match res {
+        Result::Ok(()) => {},
+        Result::Err(err) => panic!("P2MS 1-of-2 transaction validation failed: {}", err),
+    }
+}
+
+// #[test]
```

Comment:
This test is currently not running, I think

Reply:
Will confirm this and get back to you